src/python/clawutil/claw_git_status.py

```diff
@@ -18,7 +18,7 @@
 repos_list = ['classic', 'amrclaw', 'clawutil', 'pyclaw', 'visclaw', 'riemann',
               'geoclaw']
 
-def make_git_status_file(outdir=os.getcwd()):
+def make_git_status_file(outdir='.'):
```

Comment:
The problem was that `os.getcwd()` was evaluated when the function was defined, not when it is called from `runclaw`, which was after using `os.chdir` to move to a different directory.